ajax/libs/lamb/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "author": { 
+    "email": "andrea.scartabelli@gmail.com",
+    "name": "Andrea Scartabelli"
+  },
+  "description": "A lightweight, and docile, JavaScript library to help embracing functional programming.", 
+  "filename": "lamb.min.js", 
+  "homepage": "https://ascartabelli.github.io/lamb/",  
+  "keywords": [ 
+    "functional programming",
+    "utility",
+    "library",
+    "server",
+    "client",
+    "browser",
+    "node.js",
+    "es5",
+    "javascript"
+  ],
+  "license": "MIT", 
+  "name": "lamb", 
+  "repository": { 
+    "type": "git",
+    "url": "https://github.com/ascartabelli/lamb.git"
+  },
+  "version": "0.51.0", 
+  "dependencies": { 
+    
+  },
+"autoupdate": { 
+    "source": "git",
+    "target": "git://github.com/ascartabelli/lamb.git",
+    "fileMap": [
+      {
+        "basePath": "",

```

Comment:
We should fetch files under `dist` in this case, so let's use `dist` as basePath.